dist/webinstaller/make-webinstaller-manifest.js

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2020  Matthew "strager" Glazar
+// See end of file for extended copyright information.
+
+const fs = require("fs");
+
+const args = process.argv.slice(2);
+const releaseVersion = process.argv[3];
+const releaseDate = process.argv[5];
+const outputFile = process.argv[7];
+
+if (!args.includes("-releaseVersion")) {
+  console.log("error: missing -releaseVersion");
+  return 0;
+}
+if (!args.includes("-releaseDate")) {
+  console.log("error: missing -releaseDate");
+  return 0;
+}
+if (!args.includes("-Out")) {
+  console.log("error: missing -Out");
+  return 0;
+}
+
+const releaseVersionRegex = /^\d+\.\d+\.\d+$/;
+if (!releaseVersionRegex.test(releaseVersion)) {
+  console.log(
+    "error: invalid -releaseVersion; must match regular expression: " +
+      releaseVersionRegex
+  );
+  return 0;
```

Comment:
Must fix: We should `process.exit(1);` or something instead. Failures should be signaled with a non-zero process exit code.